scopes/component/component/component.graphql.ts

```diff
@@ -154,9 +154,10 @@ export function componentSchema(componentExtension: ComponentMain) {
           // graphql doesn't support map types
           return component.tags.toArray().map((tag) => tag.toObject());
         },
-        aspects: (component: Component) => {
-          const aspects = component.state.aspects.serialize();
-          return aspects;
+        aspects: (component: Component, { include }: { include?: string[] }) => {
+          if (!include) return component.state.aspects.serialize();
+          const aspects = component.state.aspects.filter(include);
+          return aspects.map((aspect) => aspect.serialize());
```

Comment:
once you do the changes in the filter, you can remove this line, and instead do
`return component.state.aspects.filter(include).serialize();`